pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -1372,7 +1373,7 @@ public void testLuceneRealtimeWithSearcherManager()
     // create and open an index writer
     File indexFile = new File(INDEX_DIR.getPath() + "/realtime-test1.index");
     Directory indexDirectory = FSDirectory.open(indexFile.toPath());
-    StandardAnalyzer standardAnalyzer = new StandardAnalyzer();
+    Analyzer standardAnalyzer = new CaseAwareStandardAnalyzer();
```

Comment:
(nit) Same for other places
```suggestion
    Analyzer analyzer = new CaseAwareStandardAnalyzer();
```